src/main/java/com/redhat/syseng/soleng/rhpam/processmigration/serializer/DateSerializer.java

```diff
@@ -24,7 +24,9 @@ public void serialize(Date date, JsonGenerator generator, SerializerProvider pro
 	    throws IOException, JsonProcessingException {
 	if (date != null) {
 	    LocalDateTime localDateTime = LocalDateTime.ofInstant(date.toInstant(), ZoneOffset.UTC.normalized());
-	    generator.writeString(localDateTime.format(DateTimeFormatter.ISO_DATE_TIME));
+            //DateTimeFormatter.ISO_DATE_TIME means I need to pass in time like this: "2018-08-14T11:00:00", otherwise this exception: Text '2018-08-14 11:00:00' could not be parsed
+	    //generator.writeString(localDateTime.format(DateTimeFormatter.ISO_DATE_TIME));
+	    generator.writeString(localDateTime.format(DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss")));
```

Comment:
Stick to ISO_DATE_TIME as it is a normalised format.

Reply:
ok, rollback.